tokio-trace-serde/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "tokio-trace-serde"
+version = "0.1.0"
+authors = ["J Haigh <debugsteven@gmail.com>"]
```

Comment:
Before releasing this, we should add `"Tokio Contributors <team@tokio.rs>"` to the authors of this crate. That can wait until we're ready to publish to `crates.io` though.